File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Billing/BillingRefundRepository.cs
namespace Acme.Legacy.Billing;

public class BillingRefundRepository
{
  public string WriteRefundAuditEntry(string refundId)
  {
    string sql = "EXEC billing.rebuild_asmx_refund_audit; SELECT * FROM billing.asmx_refund_audit_entries";
    return sql;
  }
}

File: eval/fixtures/legacy-directory-backed-jaxrs-commerce/src/main/java/com/acme/legacy/billing/BillingRefundRepository.java
package com.acme.legacy.billing;

public class BillingRefundRepository {
  public RefundAuditResult writeRefundAuditEntry(String refundId) {
    String sql = "CALL rebuild_jaxrs_refund_audit(); SELECT * FROM billing.jaxrs_refund_audit_entries";
    return new RefundAuditResult(sql);
  }
}
